3p/recaptcha.js

```diff
@@ -124,28 +137,63 @@ function initializeIframeMessagingClient(window, grecaptcha, dataObject) {
  * action {string} - action to be dispatched with grecaptcha.execute
  * id {number} - id given to us by a counter in the recaptcha service
  *
+ * @param {Window} window
  * @param {*} grecaptcha
  * @param {Object} data
  */
-function actionTypeHandler(grecaptcha, data) {
-  // TODO: @torch2424: Verify message origin
-  // TODO: @torch2424: Verify sitekey is the same as original
-  const executePromise = grecaptcha.execute(data.sitekey, {
-    action: data.action,
-  });
+function actionTypeHandler(window, grecaptcha, data) {
```

Comment:
use `win` for param to not be confused with the global window object.

Reply:
Done.